src/Orleans.Core.Abstractions/Streams/Core/IAsyncBatchObserver.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Orleans.Streams
+{
+    public class OrderedItem<T>
```

Comment:
Ordered is a bit weird naming for me here, since this is the only place has "Ordered" keyword , among IAsyncObserver/Observerble or BatchObserver/Observable interfaces

I kinda like the name Kyle suggested "StreamItem"

Reply:
The concern is that the introduction of the sequence token added ordering information.  The fact that the term isn't used anywhere doesn't negate this.  I'll think more on this and try and find a better way of expressing it, but I do think it needs be clear to users.